ols/app/endpoints/health.py

```diff
@@ -8,12 +13,22 @@
 
 
 @router.get("/readiness")
-def readiness_probe() -> dict[str, str]:
+def readiness_probe_get_method() -> dict[str, str]:
     """Ready status of service."""
     return {"status": "1"}
 
 
 @router.get("/liveness")
-def liveness_probe() -> dict[str, str]:
+def liveness_probe_get_method() -> dict[str, str]:
     """Live status of service."""
     return {"status": "1"}
+
+
+@router.head("/readiness")
+def readiness_probe_head_method() -> None:
+    """Ready status of service."""
+
```

Comment:
It would be better to return None. This might confuse someone.

Reply:
yup